bundles/org.openhab.binding.radiothermostat/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,238 @@
+				<label>Disable Retrieval of Run-time Data</label>
+				<description>Optional Flag to Disable the Retrieval of Run-time Data from the Thermostat</description>
+				<default>0</default>
+			</parameter>
+			<parameter name="disableHumidity" type="integer" min="0" max="1" required="false">
+				<label>Disable Retrieval of Humidity Data</label>
+				<description>Optional Flag to Disable the Retrieval of Humidity Data from the Thermostat</description>
+				<default>0</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="name" advanced="true">
+		<item-type>String</item-type>
+		<label>Thermostat Name</label>
+		<description>The Name of the Thermostat</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="model" advanced="true">
+		<item-type>String</item-type>
+		<label>Thermostat Model</label>
+		<description>The Model Number and Firmware Version of the Thermostat</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="temp-temperature">
+		<item-type>Number</item-type>
+		<label>Temperature</label>
+		<description>The Current Temperature Reading of the Thermostat</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number</item-type>
+		<label>Humidity</label>
+		<description>The Current Humidity Reading of the Thermostat</description>
+		<category>Humidity</category>
+		<state readOnly="true" pattern="%d %unit%" />
+	</channel-type>
+
+	<channel-type id="mode">
+		<item-type>Number</item-type>
+		<label>Mode</label>
+		<description>The Current Operating Mode of the HVAC System</description>
+		<state>
+			<options>
+				<option value="0">Off</option>
+				<option value="1">Heat</option>
+				<option value="2">Cool</option>
+				<option value="3">Auto</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fan_mode">
+		<item-type>Number</item-type>
+		<label>Fan Mode</label>
+		<description>The Current Operating Mode of the Fan</description>
+	</channel-type>
+
+	<channel-type id="program_mode" advanced="true">
+		<item-type>Number</item-type>
+		<label>Program Mode</label>
+		<description>The Program Schedule That the Thermostat Is Running</description>
+		<state>
+			<options>
+				<option value="-1">None</option>
+				<option value="0">Program A</option>
+				<option value="1">Program B</option>
+				<option value="2">Vacation</option>
+				<option value="3">Holiday</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="temp-sp">
+		<item-type>Number</item-type>
+		<label>Setpoint</label>
+		<description>The Current Temperature Set Point of the Thermostat</description>
+		<category>Temperature</category>
+		<state min="35" max="95" pattern="%d" />
+	</channel-type>
+
+	<channel-type id="override">
+		<item-type>Number</item-type>
```

Comment:
should this be a switch instead?

Reply:
It is read only and cannot be turned off. The transform examples format the 0/1 for display in the UI.